apps/sms/services/js/moz_mobile_message/messaging_service.js

```diff
@@ -0,0 +1,193 @@
+/*global BridgeServiceMixin, Settings */
+'use strict';
+
+if (!('BridgeServiceMixin' in self)) {
+  importScripts('/services/js/bridge_service_mixin.js');
```

Comment:
> My current thought is that we will never call importScripts in tests because the necessary object will always be there before.

ah, nice catch, you're right!